kolibri/plugins/demo_server/assets/src/DemoServerBannerContent.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+
+  <h1 class='banner-header' v-if="bannerClosed">
+    {{ $tr('demoServerHeader') }}
+  </h1>
+  <div v-else>
+    <h1>{{ $tr('demoServerHeader') }}</h1>
+    <p>{{ $tr('demoServerP1') }}</p>
+    <ul>
+      <li>{{ $tr('demoServerL1', {user: 'learnerdemo', pass: 'pass'} ) }}</li>
+      <li>{{ $tr('demoServerL2', {user: 'coachdemo', pass: 'pass'} ) }}</li>
+      <li>{{ $tr('demoServerL3', {user: 'admindemo', pass: 'pass'} ) }}</li>
+    </ul>
+    <p>{{ $tr('demoServerP2') }}</p>
+    <p>{{ $tr('demoServerP3') }}</p>
+    <ul>
+      <li>
+        <KExternalLink
+          href="https://learningequality.org/documentation/"
+          :text="$tr('demoServerA1')"
+          target="_blank"
+        />
+      </li>
+      <li>
+        <KExternalLink
+          href="https://learningequality.org/download/"
+          :text="$tr('demoServerA2')"
+          target="_blank"
+        />
+      </li>
+    </ul>
+  </div>
```

Comment:
Won't necessarily help here, but just a heads-up though that vue has a clever nested `<template>` feature and you can do things like:

```html
<template>
  <div>
    <h1>always shown</h1>
    <template v-if="condition">
      <p>conditionally</p>
      <p>shown</p>
    </template>
  </div>
<template>
```